bitnami/nginx/Chart.yaml

```diff
@@ -25,4 +25,4 @@ name: nginx
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/nginx
   - https://www.nginx.org
-version: 13.2.13
+version: 13.2.14
```

Comment:
Could you please bump version to `13.2.15`? Version 13.2.14 was already released.

Reply:
Ok, done.